user-config.ts

```diff
@@ -0,0 +1,149 @@
+export default {
+    // ================== BASIC ==================
+    NAME: 'NODE_NAME',
+
+    // Traiding pairs
+    PAIRS: {
+        'BTC-USDC': {
+            FEE: 0.01,
```

Comment:
if SLIPPAGE and fee are equal here it means that you are adding 0.1%   spread/fee and min spread/fee is also equal to 0.01%. This is misleading the name of the property should be MIN_ACCEPTED_FEE/ MINIMUM_ACCEPTED_SPREAD. If we keep the name of the property slippage, then the logic in validators should be changed. Otherwise the name is misleading

Reply:
It's just an example value. The user should configure this anyway.